hw5/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>OOP Homework</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+<div id="root"></div>
+</body>
+<script src="oop/base_classes.js"></script>
+<script src="oop/components.js"></script>
+<script src="oop/app.js"></script>
```

Comment:
Since you haven't implemented OOP style solution, there is no need to import these files.